### PR TITLE
[docs] documented version without OpenMP support

### PR DESCRIPTION
**Pull request by StrikerRUS**
Refer to #1780.

### CODE REVIEW

File: docs/Installation-Guide.rst [hunk 1]
```diff
@@ -177,6 +179,151 @@ Docker
...
+- **Visual Studio**;
+
+- **CMake** and **VS Build Tools**;
+
+- **CMake** and **MinGW**.
+
+Visual Studio (or VS Build Tools)
+*********************************
+
+With GUI
```
Comment (StrikerRUS) [inline docs/Installation-Guide.rst]: Unfortunately, [I can't check](https://github.com/Microsoft/LightGBM/pull/1660#discussion_r216440029) whether it's enough just to uncheck OpenMP property. Can anyone test this, please?

**Comment by StrikerRUS** [comment]
Should we relax CMake verion for OpenMP-less LightGBM?

File: docs/Installation-Guide.rst (hunk 1 above)
Comment (guolinke) [reply docs/Installation-Guide.rst]: @StrikerRUS I just check it. It can work if merged https://github.com/Microsoft/LightGBM/tree/nomp-err

Comment (guolinke) [reply docs/Installation-Guide.rst]: create a new PR #1813 to fix that first

**Comment by guolinke** [comment]
@StrikerRUS I think we can just let it be, as only mac os needs higher version to support openmp, and most macos users prefer newer versions.

File: docs/Installation-Guide.rst (hunk 1 above)
Comment (StrikerRUS) [reply docs/Installation-Guide.rst]: @guolinke Great! Thanks!

**Comment by StrikerRUS** [comment]
@guolinke Agree.

**Comment by StrikerRUS** [comment]
Should OpenMP-less version be checked at CI side? Or, since it's not recommended version, we may not waste CI time?

**Comment by Laurae2** [comment]
@StrikerRUS If `nthread=1` then OpenMP is ignored when running a LightGBM model.

**Comment by StrikerRUS** [comment]
@Laurae2 Sorry, I was unclear. I mean **compilation** of the OpenMP-less version for the protection from cases like one was fixed in #1813.

**Pull request merged**